tsconfig.json

```diff
@@ -21,5 +21,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true
   },
-  "include": ["src"]
+  "include": [
+    "src",
+    "e2e"
```

Comment:
This tsconfig will also end up in user's code and we don't want e2e here I think?

Reply:
@WoLewicki, without including here `e2e` and `__tests__/e2e/GeneralSvgRenderingTest.spec.tsx` we have problems inside those folders with typescript.